### PR TITLE
for: refresh

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by mfrw — APPROVED** [review]
Although, this is pretty verbose, but it's pretty helpful :)
Thank you @EmilySeville7cfg 
LGTM

File: pages/common/for.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+> Conditionally perform a command several times.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Looping-Constructs>.
 
-- Perform a command with different arguments:
+- Execute the specified commands for the specified set:
 
-`for argument in 1 2 3; do {{command $argument}}; done`
+`for {{variable}} in {{item_a item_b item_c}}; do {{echo "Loop is executed"}}; done`
+
+- Iterate over range:
```
Comment (sbrl) [inline pages/common/for.md:10]: Perhaps this will read better?
```suggestion
- Iterate over a range of numbers:
```

File: pages/common/for.md [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+
+`for {{variable}} in {{{from}}..{{to}}..{{step}}}; do {{echo "Loop is executed"}}; done`
+
+- Iterate over files:
```
Comment (sbrl) [inline pages/common/for.md:14]: Perhaps this will read better?
```suggestion
- Iterate over a list of files:
```

File: pages/common/for.md [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+
+`for {{variable}} in {{file_a.ext file_b.ext file_c.ext}}; do {{echo "Loop is executed"}}; done`
+
+- Iterate over directories:
```
Comment (sbrl) [inline pages/common/for.md:18]: Perhaps this will read better?
```suggestion
- Iterate over a list of directories:
```

**Review by sbrl — COMMENTED** [review]
Thanks for the page edit! I've left some comments below for you to review.

File: pages/common/for.md [hunk 4]
```diff
@@ -1,12 +1,24 @@
 # for
 
-> Shell loop over parameters.
-> More information: <https://man.archlinux.org/man/for.n>.
+> Conditionally perform a command several times.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Looping-Constructs>.
 
-- Perform a command with different arguments:
+- Execute given commands for the specified set:
```
Comment (pixelcmtd) [inline pages/common/for.md:6]: I really don't like this phrasing but I can't come up with a better one either.

**Review by pixelcmtd — APPROVED** [review]

File: pages/common/for.md (hunk 4 above)
Comment (EmilySeville7cfg) [reply pages/common/for.md:6]:
```suggestion
- Execute given commands for the specified items:
```

What do you think about this variant?

Comment (pixelcmtd) [reply pages/common/for.md:6]:
```suggestion
- Execute the given commands for each of the specified items:
```

This is the best I can come up with.

File: pages/common/for.md [hunk 5]
```diff
@@ -1,12 +1,24 @@
 # for
 
-> Shell loop over parameters.
-> More information: <https://man.archlinux.org/man/for.n>.
+> Conditionally perform a command several times.
```
Comment (navarroaxel) [inline pages/common/for.md:3]: Where is the `condition`? :thinking:

File: pages/common/for.md (hunk 4 above)
Comment (navarroaxel) [reply pages/common/for.md:6]: @EmilySeville7cfg could you apply @pixelcmtd suggestion?

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**